public/js/helpers/chart_helper.js

```diff
@@ -230,3 +230,18 @@ export function synchronize (dygraphs, syncOptions) {
     })
   })
 }
+
+export function nightModeOptions (nightModeOn) {
+  if (nightModeOn) {
+    return {
+      rangeSelectorAlpha: 0.3,
+      gridLineColor: '#596D81',
+      colors: ['#2DD8A3', '#2970FF', '#FFC84E']
+    }
+  }
+  return {
+    rangeSelectorAlpha: 0.4,
+    gridLineColor: '#C4CBD2',
+    colors: ['#2970FF', '#006600', '#FF0090']
+  }
+}
```

Comment:
These will likely be very specific to each chart, so until you see a case for reuse, I'd move these to *attackcost_controller.js* so you don't need the import.